assets/resources/equipment/weapons/teleporter.geq

```diff
@@ -0,0 +1,11 @@
+{
+  "Name": "Equipment",
+  "Description": "Equipment definition.",
+  "Icon": "",
+  "Type": "Weapon",
```

Comment:
Type should be Tool

Reply:
Fixed in https://github.com/apetavern/grubs/pull/355/commits/5359663d0cb61be10212e386b98cd55759187664